cli/vm/cli.go

```diff
@@ -782,8 +796,8 @@ func handleLoadGo(c *cli.Context) error {
 		return fmt.Errorf("can't create manifest: %w", err)
 	}
 	var signers []transaction.Signer
-	if len(args) > 1 {
-		signers, err = cmdargs.ParseSigners(args[1:])
+	if len(args) > 2 && args[1] == cmdargs.CosignersSeparator {
```

Comment:
`else` what? I'd say we have some excessive parameters that we can't use and there should be an error printed.